mitodl/micromasters
# Set default value of account_privacy set to 'public to other MicroMaster’s students'

#### What are the relevant tickets?
Resolves https://github.com/mitodl/micromasters/issues/484

#### What's this PR do?
- Set default value of account_privacy set to 'public to other MicroMaster’s students'

#### Where should the reviewer start?
- see migration 0016
- see profiles/model

#### How should this be manually tested?
- Login MM with new user, you can see ```api/v0/profiles/NewUser/``` or open ```/profile``` go to privacy tab.
@pdpinch @aliceriot @noisecapella 

#### Screenshots (if appropriate)
<img width="1092" alt="screen shot 2016-06-09 at 6 46 41 pm" src="https://cloud.githubusercontent.com/assets/10431250/15931785/8edc403a-2e72-11e6-856e-1678b5a14f1a.png">
